doc/release-notes/0.31/0.31.md

```diff
@@ -48,6 +48,13 @@ The following table covers notable changes in v0.31.0. Further information about
 </thead>
 <tbody>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/3627">#3267</a></td>
+<td valign="top">New <tt>-XX:[+|-]ShowHiddenFrames</tt> option added</td>
+<td valign="top">All versions</td>
+<td valign="top">This option controls if Hidden method frames JVM generated for MethodHandles are displayed in stacktrace. This option is disabled by default and can be enabled for debugging purposes.</td>
```

Comment:
`Hidden` doesn't need to be capitalized.
You can remove `JVM`.
`stacktrace` should be `a stacktrace`.